PRIVACY.md

```diff
@@ -0,0 +1,17 @@
+# Privacy Notes
+
+## LSP server binary
+
+The LSP server performs no network access.
```

Comment:
Maybe it's worth mentioning that the `rust-analyzer` binary will run build scripts (as part of cargo check) and proc macros (as part of macro expansion), and they can access network or otherwise affect privacy.

Reply:
Umm..

Reply:
Oh, sorry